packages/website/docs/createRecentSearchesPlugin.md

```diff
@@ -2,9 +2,37 @@
 id: createRecentSearchesPlugin
 ---
 
+The Recent Searches plugin displays a list of the latest searches the user made on your autocomplete's empty screen.
```

Comment:
I understand this sentence as if it only display the searches made on the empty screen. It's not necessarily displayed on the empty screen neither.

```suggestion
The Recent Searches plugin displays a list of the latest searches the user made.
```